catalogue/webapp/services/catalogue/works.js

```diff
@@ -79,5 +80,19 @@ export async function getWork({
   }
 
   const json = await res.json();
-  return json;
+
+  const iiifPresentationLocation = getIiifPresentationLocation(json);
+
+  if (iiifPresentationLocation) {
+    const iiifManifest = await fetch(iiifPresentationLocation.url);
+    return {
+      iiifManifest: await iiifManifest.json(),
+      ...json,
+    };
+  } else {
+    return {
+      iiifManifest: null,
+      ...json,
+    };
```

Comment:
It get's confusing when we merge objects like this and creates implicit magic that we have to know about further down the pipe.

Another option would be to keep this closer to the edge and do it in getInitialProps - thoughts?